prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      var fullname = 'Anthony Pineda';
+      console.log(fullname);
+      var numberofpets = '0';
```

Comment:
Here you should leave out the quotation marks so that JavaScript knows that `numberOfPets` should be a number and not a string.